lib/rest2.js

```diff
@@ -619,6 +620,22 @@ class RESTv2 {
     }, cb, Trade)
   }
 
+  /**
+   * @param {number} start
+   * @param {number} end
+   * @param {number} limit
+   * @param {Method} cb
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-logins-hist
+   */
+  logins (start = 0, end = Date.now(), limit = 25, cb) {
```

Comment:
Could you copy the same style as `candles` and use an object as the argument please?